### DISCUSSION TITLE
Inesperado

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#477] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/inesperado/

**Comment by one-eyed-willy** [comment]
Santo reboco! A mina é tipo o Chucky sem maquiagem :rofl:

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [terça-feira, 31 de maio de 2022](<https://www.vish.blog.br/videos/inesperado/#comment_1142387119>):

> Santo reboco! A mina é tipo o Chucky sem maquiagem 🤣

Mas alguns asiáticos usam MUITA maquiagem mesmo
